src/prop/cnf_stream.cpp

```diff
@@ -186,6 +186,7 @@ SatLiteral CnfStream::newLiteral(TNode node,
     {
       Trace("cnf") << d_name << "::newLiteral: new var\n";
       lit = SatLiteral(d_satSolver->newVar(isTheoryAtom, canEliminate));
+      d_stats.d_numAtoms++;
```

Comment:
Do you want to count all or only the theory atoms?

Reply:
Counting all atoms is what I intended, which I believe this is doing?

Reply:
Yes, just wanted to make sure.